### PR TITLE
INTYGFV-14452: Bugfixes footer and header

### PR DESCRIPTION
**Pull request by HelenaEdblad**

### CODE REVIEW

File: packages/common/src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -95,19 +95,65 @@ export const Footer: React.FC = () => (
...
+                <p>Väljer du att inte acceptera kakor så kan du inte identifiera dig med e-legitimation i denna e-tjänst.</p>
+                <p>
+                  Mer information om kakor kan du finna på{' '}
+                  <a href="https://pts.se/sv/privat/internet/integritet/kakor-cookies/" target="_blank">
+                    Kommunikationsmyndigheten PTS sida om kakor launch
+                    <ExternalLinkIconImg src={externalLinkIcon} />
+                  </a>
+                  .
+                </p>
+              </TextWithInfoModal>
```
Comment (mhornfeldt) [inline packages/common/src/components/Footer/Footer.tsx:155]: This could be extracted to its own component.

File: yarn.lock [hunk 2]
```diff
@@ -10374,8 +10374,8 @@ indexes-of@^1.0.1:
   integrity sha1-8w9xbI4r00bHtn0985FVZqfAVgc=
 
 "inera-core-css@git+https://bitbucket.org/ineraservices/inera-core-css.git#master":
-  version "0.16.0"
-  resolved "git+https://bitbucket.org/ineraservices/inera-core-css.git#75f597de378ebb76d95ed2d5b4ecd649d8e57bf2"
+  version "0.17.0"
```
Comment (mhornfeldt) [inline yarn.lock:10377]: There was issues with this version, the pipeline failed after updating last time.

**Review by mhornfeldt — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 2 above)
Comment (HelenaEdblad) [reply yarn.lock:10377]: Ok, I fix!

File: yarn.lock [hunk 3]
```diff
@@ -10375,7 +10375,7 @@ indexes-of@^1.0.1:
 
 "inera-core-css@git+https://bitbucket.org/ineraservices/inera-core-css.git#master":
   version "0.16.0"
-  resolved "git+https://bitbucket.org/ineraservices/inera-core-css.git#75f597de378ebb76d95ed2d5b4ecd649d8e57bf2"
+  resolved "git+https://bitbucket.org/ineraservices/inera-core-css.git#97c06011b5453c826040b13bfc4257fdf889d61c"
```
Comment (mhornfeldt) [inline yarn.lock:10378]: Maybe this should be changed back too?

**Review by mhornfeldt — APPROVED** [review]

File: yarn.lock (hunk 3 above)
Comment (HelenaEdblad) [reply yarn.lock:10378]: Oh, of course.

**Pull request merged**